docs/build/sdks/python/assets/index.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Pieces Wrapper Python SDK Assets
+description: Learn how to set up and use the Pieces Wrapper Python SDK Asset methods
+---
+
+# Pieces wrapper python SDK asset methods
+
+## create_asset()
+
+```python
+create_asset(content, metadata)
+```
+
+Creates a new asset.
+
+### Parameters
+
+Name | Type | Notes
+------------- | ------------ | -------------
+ **content** | **string**| [required]
+ **metadata** | **FragmentMetadata**| [optional]
+
+### Example
+
+```python
+from pieces_os_client.wrapper import PiecesClient
+from pieces_os_client import FragmentMetadata
+
+# Replace 'your_base_url' with the base URL of your Pieces OS server
+pieces_client = PiecesClient(config={'baseUrl': 'your_base_url'})
```

Comment:
Just small update here @choir27 we change the config from baseUrl to host

```diff
-pieces_client = PiecesClient(config={'baseUrl':'your_base_url'})
+pieces_client = PiecesClient(config={'host':'your_base_url'})
```
It should be updated everywhere too